tests/unit/test_nav.py

```diff
@@ -8,7 +8,7 @@ def predicate_false(request):
 def test_iter_nav(rf):
     items = [
         nav.NavItem("Workspace", "workspace_index"),
-        nav.NavItem("Requests", "request_index", predicate_false),
+        nav.NavItem("Requests", "requests_for_researcher", predicate_false),
```

Comment:
it would be nice to also add a specific nav test for the logged in output checker's nav button - you should be able to extend this current nav test, re-using something from the `test_request.py` to log in?